README.md

```diff
@@ -1,53 +1,162 @@
 
-Taxonomies for products and languages: https://review.docs.microsoft.com/new-hope/information-architecture/metadata/taxonomies?branch=master
--->
+* HTTP-triggered function: The example demonstrates how to pass in query string parameters to the function.
+* Timer-triggered function: A non-HTTP triggered function (in the form of a timer-triggered function) is included to demonstrate how to test a function that is not callable via a standard HTTP request.
 
-Give a short description for your sample here. What does it do and why is it important?
+![Testing Azure Functions with JavaScript in VS Code](./media/azure-functions-test-vs-code-mocha.png)
 
-## Contents
+To read more about the testing Azure functions, read [Strategies for testing your code in Azure Functions](https://docs.microsoft.com/azure/azure-functions/functions-test-a-function)
 
-Outline the file contents of the repository. It helps users navigate the codebase, build configuration and any related assets.
 
-| File/folder       | Description                                |
-|-------------------|--------------------------------------------|
-| `src`             | Sample source code.                        |
-| `.gitignore`      | Define what to ignore at commit time.      |
-| `CHANGELOG.md`    | List of changes to the sample.             |
-| `CONTRIBUTING.md` | Guidelines for contributing to the sample. |
-| `README.md`       | This README file.                          |
-| `LICENSE`         | The license for the sample.                |
+## Setup
 
-## Prerequisites
+This procedure uses the [VS Code Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) to create Azure Functions
 
-Outline the required components and tools that a user might need to have on their machine in order to run the sample. This can be anything from frameworks, SDKs, OS versions or IDE releases.
+To set up your environment, initialize a new Node.js app in an empty folder by running `npm init`.
 
-## Setup
+```bash
+npm init -y
+```
+
+Next, install Jest by running the following command:
+
+```bash
+npm i mocha chai sinon
+```
+
+Now update _package.json_ to replace the existing test command with the following command:
+
+```bash
+"scripts": {
+    "test": "mocha --recursive '**/tests/*'"
+}
+```
+
+### Create test modules
+
+With the project initialized, you can create the modules used to run the automated tests.
+
+Begin by creating a new Azure function using the VS Code Functions extension to [create a new JavaScript HTTP Function](/azure/javascript/tutorial-vscode-serverless-node-01) and name it *HttpTrigger*. Create new folders named *testing* and *test* to hold the support modules & test cases.

```

Comment:
```suggestion
Begin by creating a new function using the VS Code Functions extension to [create a new JavaScript HTTP Function](https://docs.microsoft.com/azure/javascript/tutorial-vscode-serverless-node-01) and name it *HttpTrigger*. Create new folders named *testing* and *test* to hold the support modules & test cases.
```
This change helps adhere to branding better. An Azure Functions app has functions, but there is no such thing as an "Azure function" 😉